public/main.js

```diff
@@ -67,3 +69,107 @@ artistName.addEventListener('keypress', function (e) {
       getArtistName();
   }
 });
+
+
+var list = document.getElementById('list');
+var showID = list.addEventListener('click', myClick, false);
+
+var showAlTracks = list.addEventListener('click', userClick, false);
+
+function myClick() {
+  if(event.target.id.length === 22) {
+    const thenable = fetch('/albums/' + event.target.id);
+    while (list.firstChild) {
+    list.removeChild(list.firstChild);
+  }
+    thenable
+      .then(result => result.json())
+      .then(showAlbums)
+      .then(results => console.log(results))
+      .catch(error => console.error(error));
+    console.log(event.target.id);
+  }
+}
+
+var showAlbums = (results) => {
+  var albumList = document.getElementById('list')
+
+  for (let i = 0; i < results.length; i ++) {
+    var obj = results[i];
+    var $artist = document.createElement('div');
+    var $info = document.createElement('div')
+    var $albumName = document.createElement('p');
+    var $artistName = document.createElement('p');
+    var $type = document.createElement('p');
+    var $img = document.createElement('img');
+
+    $albumName.textContent = obj.name;
+    $artistName.textContent = 'Artist: ' + obj.artists[0].name;
+    $type.textContent = 'Type: ' + obj.album_type.toUpperCase();
+
+    $artist.setAttribute('class', 'artistInfo');
+    $artist.setAttribute('id', 'removeList');
+    $img.setAttribute('src', obj.images[0].url);
+    $img.setAttribute('class', 'artistImg');
+    $albumName.setAttribute('class', 'albumName');
+    $albumName.setAttribute('id', obj.id);
+
+    $artist.appendChild($img);
+    $artist.appendChild($info);
+    $info.appendChild($albumName);
+    $info.appendChild($type);
+    $info.appendChild($artistName);
+    albumList.appendChild($artist);
+  }
+  return results;
+};
+
+
+function userClick() {
```

Comment:
Same here, try to make a more meaningful name that describes with the function does.